libraries/botbuilder/tests/streaming/botFrameworkAdapterStreaming.test.js

```diff
@@ -122,13 +122,50 @@ describe('BotFrameworkAdapter Streaming tests', () => {
                 await bot.run(context);
                 throw new Error('useWebSocket should have thrown an error');
             }).catch(err => {
-                expect(err.message).to.equal('Unauthorized. No valid identity.');
-                const socketResponse = MockNetSocket.createNonSuccessResponse(401);
+                expect(err.message).to.equal('Unauthorized. Is not authenticated');
+                const socketResponse = MockNetSocket.createNonSuccessResponse(401, err.message);
                 expect(writeSpy.called).to.be.true;
                 expect(writeSpy.calledWithExactly(socketResponse)).to.be.true;
                 expect(destroySpy.calledOnceWithExactly()).to.be.true;
             });
         });
+
+        it('returns status code 400 when request is missing Authorization header', async () => {
+            const bot = new ActivityHandler();
+            settings = new TestAdapterSettings('appId', 'password');
+            const adapter = new BotFrameworkAdapter(settings);
+            const requestWithoutAuthHeader = new MockHttpRequest();
+            
+            const socket = new MockNetSocket();
+            const writeSpy = spy(socket, 'write');
+            const destroySpy = spy(socket, 'destroy');
+
+            await adapter.useWebSocket(requestWithoutAuthHeader, socket, Buffer.from([]), async (context) => {
+                await bot.run(context);
+                
+                throw new Error('useWebSocket should have thrown an error');
+            }).catch(err => {
```

Comment:
wouldn't it be nicer to avoid mix-n-match explicit promise code and await code. i.e. why not use try { } catch(err) { }

Reply:
"why not use try { } catch(err) { }" 
Because the copy-pasta of the previously-written unit test didn't have that :stuck_out_tongue_winking_eye:

Changed it to use trycatch now!